### ISSUE TITLE
Add a menu option "Open new window" to launch icon on macOS

### ISSUE THREAD

**Issue by dariuspranskus** [defold/defold#4863]
Add a menu option "Open new window" to launch icon on MAC OS, which would open a new editor window where user can open, for example a demo project.

**Comment by subsoap** [comment]
AFAIK MacOS has file system issues when opening more than one copy of the editor currently?

**Comment by britzl** [comment]
> MacOS has file system issues when opening more than one copy of the editor currently

I don't think so. Maybe? I sometimes have more than one editor running, but never for longer periods of time so I may not have encountered the problem you are thinking of.
